app/Http/Resources/PermissionResource.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I implemented this to start to uniform some generic error outputs rather that rely on laravels default JSON responses which are not always uniformed. However I ran out of time to implement this.